documentation/src/getting_started.md

```diff
@@ -65,6 +46,7 @@ graph LR
   - Live view: Supported directly by gPhoto2, or using HDMI out and a USB HDMI capture device (which can act like a webcam)
   - Capturing: Supported directly by gPhoto2
 - Optional: Photo printer
+- Optional: Internet connection
```

Comment:
I agree that it's optional. We do not have a setting as of yet to hide the make QR code thing, or proper error handling when it does not succeed...